### ISSUE TITLE
Creates separate diagram file for each call in try...catch block

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ankit4u3/jtracert#8]
```
What steps will reproduce the problem?
1. write few calls in try...catch block for first time and second time
remove the try...block.

What is the expected output? What do you see instead?
Creates 1 file per call in try...catch block in first case. In second case
without try...catch block, one file for entire trace.

What version of the product are you using? On what operating system? What
JRE are you using (vendor and version) ?
JDK/JRE 1.6, jTracert 0.0.4

Please provide any additional information below.


```

Original issue reported on code.google.com by `dhaval.b.nagar` on 31 Jan 2009 at 8:42

**Comment by GoogleCodeExporter** [comment]
```
// CASE1 Main.java: Main method contains these four statements.
try{
   JMSManager manager = new JMSManager();
   manager.createConnection();
   manager.sendMessage("hello, world");
   manager.close();
}catch(Exception e){e.printStackTrace();}

run with following parameters:
-DanalyzerOutput=sequenceFileSystem -DoutputFolder=c:\tmp\sddig
-javaagent:c:\jar\jTracert.jar

output files:
JMSManager.init.sq
   test/jms/JMSManager.init {
   }
JMSManager.createConnection.sq, 
   test/jms/JMSManager.createConnection {
   }
JMSManager.sendMessage.sq, 
   test/jms/JMSManager.sendMessage {
   }
JMSManager.close.sq
   test/jms/JMSManager.close {
   }


// CASE2 Main.java: Main method contains these four statements (try...catch 
removed).
   JMSManager manager = new JMSManager();
   manager.createConnection();
   manager.sendMessage("hello, world");
   manager.close();

run with following parameters:
-DanalyzerOutput=sequenceFileSystem -DoutputFolder=c:\tmp\sddig
-javaagent:c:\jar\jTracert.jar

output files:
Main.main.sq
    test/jms/Main.main {
      test/jms/JMSManager.init {
      }
      test/jms/JMSManager.createConnection {
      }
      test/jms/JMSManager.sendMessage {
      }
      test/jms/JMSManager.close {
      }
    } 

```

Original comment by `dhaval.b.nagar` on 31 Jan 2009 at 8:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `Dmitry.Bedrin` on 31 Jan 2009 at 9:22
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
A sample application (called Issue8) was created and added to the SVN - I didn't
manage to reproduce this defect.

Can you please provide me with your sources (classes from test.jms package), so 
I
could make more tests.

You can attach the sources directly to this issue, or send me them via email:
Dmitry.Bedrin@gmail.com

Thanks,
 Dmitry.
```

Original comment by `Dmitry.Bedrin` on 4 Feb 2009 at 9:23
- Changed state: **Postponed**

**Comment by GoogleCodeExporter** [comment]
```
i have tested again and i am sending u the src and the demo results.
```

Original comment by `dhaval.b.nagar` on 4 Feb 2009 at 1:09

Attachments:
- [tracetest.rar](https://storage.googleapis.com/google-code-attachments/jtracert/issue-8/comment-4/tracetest.rar)

**Comment by GoogleCodeExporter** [comment]
```
Thanks - I can reproduce this defect also!

Status updated to 'Accepted', to be fixed in release 0.0.5
```

Original comment by `Dmitry.Bedrin` on 4 Feb 2009 at 3:23
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
It appears that the JVM version is the case.

If you amend build.xml in your example and add target="1.5" attribute to the 
javac
task, everything would work as expected.

I'm investigating why it happens now.
```

Original comment by `Dmitry.Bedrin` on 5 Feb 2009 at 10:03

**Comment by GoogleCodeExporter** [comment]
```
Fixed in SVN revision r118 .
This fix will be delivered in next build.
```

Original comment by `Dmitry.Bedrin` on 6 Feb 2009 at 12:33
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
done. i build it with new revision and it is working fine.
```

Original comment by `dhaval.b.nagar` on 6 Feb 2009 at 4:45

**Comment by GoogleCodeExporter** [comment]
```
Fixed in jTracert version 0.0.5
```

Original comment by `Dmitry.Bedrin` on 4 Mar 2009 at 1:36
- Changed state: **Verified**
